### PR TITLE
Import assets to dist/ rather than imported-assets/

### PR DESCRIPTION
**Pull request by howard-wallis**
Stop worrying about filename conflicts and copy assets to `dist/` rather than deep down in `dist/imported-assets/{pkg.name}/dist`, solving the awkward issue we were discussing on https://github.com/justeat/fozzie/pull/21
This makes the assumption that all modules that provide assets will be doing so from a folder named `dist`.

### CODE REVIEW

File: tasks/copy.js [hunk 1]
```diff
@@ -92,17 +93,17 @@ gulp.task('copy:fonts', () => {
...
+            filepath,
             name: split[split.length - 2]
         };
     };
 
     const copyFromPackage = pkg => new Promise((resolve, reject) => {
-        gutil.log(`❯❯ Copying any assets in ${pkg.path} to ${pathBuilder.importedAssetsDistDir}/${pkg.name}`);
-        copyAssets(pkg.path, `${pathBuilder.importedAssetsDistDir}/${pkg.name}`, err => {
+        gutil.log(`❯❯ Copying any assets from ${pkg.name} to ${path.dirname(config.assetDistDir)}`);
+        copyAssets(`${pkg.filepath}`, `${path.dirname(config.assetDistDir)}`, err => {
```
Comment (howard-wallis) [inline tasks/copy.js]: I'm using path.dirname here to navigate up one directory from `dist/` so that `copyAssets` merges the current `dist/` with the one from the package. This also covers the use of `config.assetDistDir = 'assets/dist/` as seen in globalweb and the docs site.

File: tasks/css.js [hunk 2]
```diff
@@ -117,7 +118,7 @@ gulp.task('css:bundle', () => gulp.src(`${pathBuilder.scssSrcDir}/**/*.scss`)
         postcss([
             // Converts any specified assets to data URIs
             assets({
-                loadPaths: [pathBuilder.imgSrcDir, pathBuilder.importedAssetsDistDir]
+                loadPaths: [pathBuilder.imgSrcDir, config.assetDistDir, path.dirname(config.assetDistDir)]
```
Comment (DamianMullins) [inline tasks/css.js]: Do you need both `config.assetDistDir` and `path.dirname(config.assetDistDir)` here?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@justeat/gulp-build-fozzie",
-  "version": "5.0.0",
+  "version": "5.1.0",
```
Comment (DamianMullins) [inline package.json]: Package version needs to be bumped again as another PR has gone in as v5.1.0

File: tasks/css.js (hunk 2 above)
Comment (howard-wallis) [inline tasks/css.js]: Nope, I'll remove `config.assetDistDir`
